env_manager/backends/venv_interface.py

```diff
@@ -51,33 +51,36 @@ def install_packages(self, environment_path, packages, channels=None, force=Fals
             executable_path = Path(environment_path) / "Scripts" / "python.exe"
         else:
             executable_path = Path(environment_path) / "bin" / "python"
-
-        result = subprocess.check_output(
-            [executable_path, "-m", "pip", "install"] + packages
-        ).decode("utf-8")
-        print(result)
-        return result.split("\r\n")
+        try:
+            command = [str(executable_path), "-m", "pip", "install"] + packages
+            result = subprocess.run(command, capture_output=True, check=True, text=True)
+            return (True, result)
+        except subprocess.CalledProcessError as error:
+            return (False, f"{error.returncode}: {error.stderr}")
```

Comment:
Perhaps here we should set the [`PIP_REQUIRE_VIRTUALENV`](https://docs.python-guide.org/dev/pip-virtualenv/) env var to be extra sure that we avoid pip leaking packages to the user installation directory in case this fails for whatever reason.